Gemfile

```diff
@@ -26,7 +26,7 @@ end
 
 # No need to run rubocop on earlier versions
 if RUBY_VERSION >= '2.4' && RUBY_ENGINE == 'ruby'
-  gem "rubocop", "~> 0.52.1"
+  gem 'rubocop', "~> 1.0", "< 1.12"
```

Comment:
I think there was a bug in RuboCop preventing it from correctly parsing older Ruby code when running on newer Ruby versions or the other way around. I'd just remove `< 1.12` and see what happens.